### PR TITLE
Added pause, resume, and volume functionality

### PR DESCRIPTION
**Pull request by thenathanyang**
Merging branch: `nathan/pause-resume-volume`

### CODE REVIEW

File: spotifyPlayer.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def __init__(self, spotify_client_id, spotify_client_secret):
 
 		self.currSongID = 1
 		self.mp3Files = []
-		self.mixer = mixer.init()
+		mixer.init()
 
 		self.tempDir = "mp3FilesTemp"
```
Comment (nkansal96) [inline spotifyPlayer.py:25]: Don't manually manage the creation and location of temporary files. Use [the built-in library](https://docs.python.org/3/library/tempfile.html).

File: spotifyPlayer.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def playMP3(fileName):
 
 		# Removes the MP3 file once song is done playing
 		def checkPlayStatus(fileName):
-			while pygame.mixer.music.get_busy():
+			while mixer.music.get_busy():
```
Comment (nkansal96) [inline spotifyPlayer.py:68]: Does this return `true` if you call`mixer.music.pause()`?

File: spotifyPlayer.py [hunk 3]
```diff
@@ -74,3 +74,19 @@ def checkPlayStatus(fileName):
...
+
+	def pause(self):
+		mixer.music.pause()
+		return True
+
+	def resume(self):
+		mixer.music.unpause()
+		return True
+
+	def volume(self, value):
```
Comment (nkansal96) [inline spotifyPlayer.py]: This isn't C, don't return `-1` or `value`. In fact, don't return anything. Just use the defaults silently and document this behavior. e.g.:

```python
vol = max(0, min(100, value)) / 100
```

File: spotifyPlayer.py [hunk 4]
```diff
@@ -74,3 +74,19 @@ def checkPlayStatus(fileName):
 		threading.Thread(target=checkPlayStatus, args=(currFileName,)).start()
 
 		return True
+
+	def pause(self):
```
Comment (nkansal96) [inline spotifyPlayer.py:78]: Why should `pause` and `resume` return anything at all if they just return `true`?

File: spotifyPlayer.py [hunk 5]
```diff
@@ -65,7 +65,7 @@ def playMP3(fileName):
 
 		# Removes the MP3 file once song is done playing
 		def checkPlayStatus(fileName):
-			while pygame.mixer.music.get_busy():
+			while mixer.music.get_busy():
 				pass
 			if (Path(fileName).is_file()):
```
Comment (nkansal96) [inline spotifyPlayer.py:70]: If you're going to delete this file, then make sure all of your state is consistent. (e.g. update `self.mp3Files`)

**Review by nkansal96 — CHANGES_REQUESTED** [review]

**Comment by nkansal96** [comment]
In [this function](https://github.com/nkansal96/music-player/blob/8cfa2ecd6ea976404fd3be8f5e2b9a9588d9107a/spotifyPlayer.py#L40), you should use [this library function](https://docs.python.org/3.5/library/shutil.html#shutil.rmtree) to just remove the whole directory tree, since you know it will only have temporary files in it.

File: spotifyPlayer.py (hunk 2 above)
Comment (thenathanyang) [reply spotifyPlayer.py:68]: Yes. `pygame.mixer.music.pause()` doesn't affect the state of `pygame.mixer.music.get_busy()`

**Pull request closed without merging**